frontend/src/components/sidebar/SidebarButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Just out of interest, what rules define import order now?

Reply:
Typescript hero unfortunately is not well documented (or supported) - but I do not know of a better extension to organise imports in TS.

Order as far as I can tell is:
* Group 1 - import module - i.e. `import 'my-module';`
* Group 2 - external import - i.e. `import X from 'react';`
* Group 3 - internal import - i.e. `import X from '../employees'`

Each group is sorted alphabetically by file/module name. 
Also within multiple named imports the items are sorted alphabetically (e.g. `import { fetchEmployee, selectEmployees } from '../../features/employeeSlice'`)